libstemmer/modules.txt

```diff
@@ -32,6 +33,7 @@ portuguese      UTF_8,ISO_8859_1        portuguese,pt,por
 romanian        UTF_8,ISO_8859_2        romanian,ro,rum,ron
 russian         UTF_8,KOI8_R            russian,ru,rus
 serbian         UTF_8                   serbian,sr,srp
+slovak          UTF_8,ISO_8859_2        slovak,sk,svk
```

Comment:
The 2 and 3 letter codes should be those specified by ISO 639:

https://en.wikipedia.org/wiki/List_of_ISO_639-1_codes

So for Slovak that should be: `slovak,sk,slk,slo` (those for Czech above are also wrong but I've opened #151 to merge the Czech stemmer since I can write that one up for the website).